ldi-core/ldes-client/tree-node-supplier/src/main/java/ldes/client/treenodesupplier/repository/filebased/FileManager.java

```diff
@@ -14,9 +14,9 @@
 public class FileManager {
 
 	public Stream<String> getRecords(String file) {
-		if (Path.of(STATE_DIRECTORY, file).toFile().exists()) {
+		if (Path.of(STATE_DIRECTORY, file).toAbsolutePath().toFile().exists()) {
```

Comment:
Maybe we can extract `Path.of(STATE_DIRECTORY, file).toAbsolutePath().toFile().exists()` so that we don't have three duplicates of this logic?

Reply:
extracted two methods:
* getAbsolutePath()
* fileExists()